server/data_access_layer/company.js

```diff
@@ -0,0 +1,17 @@
+const { base } = require('./index');
+const { extractContentFromRecords, getAllFromTable } = require('./helpers');
+
+async function getCompany(companyCode) {
+  const companyTable = base('Company Codes').select({
+    // Filter by company Code
+    filterByFormula: '{Code}=' + companyCode,
+    view: 'Grid view'
+  });
+  const company = await getAllFromTable(companyTable);
+  var companyData = { 'ID': company[0]['fields']['ID'], 'Title': company[0]['fields']['Title'] }

```

Comment:
optional:

```
const company = await getAllFromTable(companyTable)[0];
const companyData = { ID: company.fields.ID, Title: company.fields.Title };
return companyData;
```

Reply:
agreed ^ 
+ can you also split each field onto a newline so it's easier to read